code/modules/shuttle/emergency.dm

```diff
@@ -585,11 +588,37 @@
 				var/destination_dock = "emergency_away"
 				if(is_hijacked() || elimination_hijack())
 					// just double check
-					SSmapping.lazy_load_template(LAZY_TEMPLATE_KEY_NUKIEBASE)
-					destination_dock = "emergency_syndicate"
-					minor_announce("Corruption detected in \
-						shuttle navigation protocols. Please contact your \
-						supervisor.", "SYSTEM ERROR:", sound_override = 'sound/misc/announce_syndi.ogg')
+					if(hijacker.mind)
+
+						var/destination = hijacker.mind.get_desired_hijack_destination()
+
+					// To add another destination, go to the _DEFINES\antagonists to name your desitnation and give it a number to define it.
+					// Remember to add it to your antagonist's hijack_location varible as well.
+					// Then define where you want the shuttle to dock by by choosing or creating a new docking port in switch statement
+					// get_desired_hijack_destination() takes only the destination with the highest number in the case of an mind with multiple antagonist datums
+						switch(destination)
+
+							if(DEEP_SPACE) // Our hijacker is sending us deep into space, probably because they are unafiliated with the syndicate.
+								var/x = 32 // In the middle so we don't have to worry about clipping the edge of the z-level.
+								var/y = 32
+								var/z = SSmapping.empty_space.z_value // We are created a brand new isolated Z-level
```

Comment:
This code is **not** creating a brand new isolated Z-level.

If that's what you want you need to reserve one.

Reply:
Miss-understanding of how the mapping subsystem works.